src/build.d

```diff
@@ -513,6 +514,25 @@ alias checkwhitespace = memoize!(function()
     return new DependencyRef(dep);
 });
 
+alias zip = memoize!(function()
+{
+    Dependency dep;
+    with (dep)
+    {
+        name = "zip";
+        target = srcDir.buildPath("dmdsrc.zip");
+        sources = .sources.root.chain(.sources.backend, .sources.lexer,
```

Comment:
Simple array concatenation would involve less noise (but that is propably a matter of personal preference)